validator/keymanager/types.go

```diff
@@ -43,7 +43,7 @@ func (k Kind) String() string {
 	switch k {
 	case Derived:
 		return "derived"
-	case Direct:
+	case Imported:
 		return "direct"
```

Comment:
Not changed because it will break people's wallets

Reply:
If theres anytime to do this isn't it for the beta release?